src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -67,6 +68,16 @@ export default function withAnnotations<P extends object>(
             annotator.emit('annotations_active_set', id);
         };
 
+        emitRemoveEvent = (id: string) => {
+            const { annotator } = this;
+
+            if (!annotator) {
+                return;
+            }
+
+            annotator.emit('annotations_remove', id);
```

Comment:
If we get more of these, we'll probably want to invest in a factory for method -> event pairs. Not needed right now, though.